### PR TITLE
Fixing nuget error related to package icon

### PR DESCRIPTION
**Pull request by ajaygargdev**
Changed depreciated <PackageIconUrl> to <PackageIcon>

### CODE REVIEW

File: kmd-logic-citizendocuments-client.sln [hunk 1]
```diff
@@ -13,6 +13,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution", "Solution", "{59
 		build.ps1 = build.ps1
 		build.sh = build.sh
 		global.json = global.json
+		C:\Users\AA00595162\Source\Repos\KMDLoGIC\src\portal\Kmd.Logic.Portal\public\kmd-logic-icon.png = C:\Users\AA00595162\Source\Repos\KMDLoGIC\src\portal\Kmd.Logic.Portal\public\kmd-logic-icon.png
```
Comment (adamchester) [inline kmd-logic-citizendocuments-client.sln]: This doesn't look like a good place for most users to store the icon

**Review by adamchester — CHANGES_REQUESTED** [review]

**Comment by adamchester** [comment]
The icon file is missing. It needs to be added to the repository.

File: kmd-logic-citizendocuments-client.sln (hunk 1 above)
Comment (ajaygargdev) [reply kmd-logic-citizendocuments-client.sln]: Removed from here

File: src/Kmd.Logic.CitizenDocuments.Client/Kmd.Logic.CitizenDocuments.Client.csproj [hunk 2]
```diff
@@ -4,10 +4,12 @@
     <OutputType>Library</OutputType>
     <TargetFramework>netstandard2.0</TargetFramework>
     <RootNamespace>Kmd.Logic.CitizenDocuments.Client</RootNamespace>
+    <PackageIcon>kmd-logic-icon.png</PackageIcon>
```
Comment (adamchester) [inline src/Kmd.Logic.CitizenDocuments.Client/Kmd.Logic.CitizenDocuments.Client.csproj]: Would it best to put this out the `Directory.Build.props` file with the other common shared nuget properties?

Comment (ajaygargdev) [reply src/Kmd.Logic.CitizenDocuments.Client/Kmd.Logic.CitizenDocuments.Client.csproj]: Fixed

File: src/Kmd.Logic.CitizenDocuments.Client/Kmd.Logic.CitizenDocuments.Client.csproj [hunk 3]
```diff
@@ -3,11 +3,12 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <RootNamespace>Kmd.Logic.CitizenDocuments.Client</RootNamespace>
+    <RootNamespace>Kmd.Logic.CitizenDocuments.Client</RootNamespace>
```
Comment (adamchester) [inline src/Kmd.Logic.CitizenDocuments.Client/Kmd.Logic.CitizenDocuments.Client.csproj]: accidental extra spaces?

Comment (ajaygargdev) [reply src/Kmd.Logic.CitizenDocuments.Client/Kmd.Logic.CitizenDocuments.Client.csproj]: fixed

**Review by adamchester — APPROVED** [review]
LGTM

**Pull request merged**